src/riak_object.erl

```diff
@@ -936,34 +937,38 @@ binary_version(<<?MAGIC:8/integer, 1:8/integer, _/binary>>) -> v1.
 -spec from_binary(bucket(),key(),binary(),encoding()) ->
     riak_object() | {error, 'bad_object_format'}.
 
-from_binary(B,K,Obj) ->
-    from_binary(B,K,Obj,erlang).
-from_binary(_B,_K,<<131, _Rest/binary>>=ObjTerm, _) ->
+%% Keep deprecated interface around, because I have a feeling someone
+%% will still try to use it
+
+from_binary(B,K,ObjTerm, _Enc) ->
```

Comment:
You can add an "deprecated" edoc tag to this function to make it appear in generated docs.